tasks/copy_to_glacier_cumulus_translator/test/unit_tests/test_copy_to_glacier_cumulus_translator_unit.py

```diff
@@ -0,0 +1,304 @@
+"""
+Name: test_copy_to_glacier_cumulus_translator_unit.py
+
+Description:  Unit tests for copy_to_glacier_cumulus_translator.py.
+"""
+import unittest
+import uuid
+from unittest.mock import MagicMock, patch, Mock
+
+import jsonschema
+
+import copy_to_glacier_cumulus_translator
+
+
+class TestCopyToGlacierCumulusTranslatorUnit(
+    unittest.TestCase
+):  # pylint: disable-msg=too-many-instance-attributes
+    """
+    TestCopyToGlacierCumulusTranslator.
+    """
+
+    # noinspection PyPep8Naming
+    @patch("run_cumulus_task.run_cumulus_task")
+    @patch("cumulus_logger.CumulusLogger.setMetadata")
+    def test_handler_happy_path(
+        self, mock_setMetadata: MagicMock, mock_run_cumulus_task: MagicMock
+    ):
+        """
+        Basic path with all information present.
+        """
+        event = Mock()
+        context = Mock()
+        result = copy_to_glacier_cumulus_translator.handler(event, context)
+
+        mock_setMetadata.assert_called_once_with(event, context)
+        mock_run_cumulus_task.assert_called_once_with(
+            copy_to_glacier_cumulus_translator.task, event, context
+        )
+        self.assertEqual(mock_run_cumulus_task.return_value, result)
+
+    @patch("copy_to_glacier_cumulus_translator.task")
+    @patch("cumulus_logger.CumulusLogger.setMetadata")
+    def test_handler_passthrough(
+        self, mock_setMetadata: MagicMock, mock_task: MagicMock
+    ):
```

Comment:
Quick doc string on test would be good here.

Reply:
Added